src/GCodes/GCodes.cpp

```diff
@@ -3228,6 +3228,12 @@ GCodeResult GCodes::SetOrReportOffsets(GCodeBuffer &gb, const StringRef& reply)
 			}
 		}
 	}
+	else
+	{
+		String<GCODE_LENGTH> scratch;
+		gb.AppendFullCommand(scratch.GetRef());
```

Comment:
Please use StringLength100 here to reduce stack usage.

Reply:
Used a new `StringLengthLoggedCommand` being equal to `StringLength100`.